posydon/utils/common_functions.py

```diff
@@ -1545,11 +1545,14 @@ def cumulative_mass_transfer_string(cumulative_integers):
             result += "no_RLO"
         else:
             if not added_case_word:
-                result += "case"
+                result += "case_"
                 added_case_word = True
             else:
                 result += "/"
-            result += MT_CASE_TO_STR[integer]
+            if integer in MT_CASE_TO_STR:
+                result += MT_CASE_TO_STR[integer] + '1' # from star 1
+            else:
+                result += MT_CASE_TO_STR[integer-10] + '2' # from star 2
```

Comment:
I'd suggest to add a comment to the definition for MT_CASE_TO_STR, that only single digit values are allowed in the keys.
I'm wondering, whether it would be better instead of using a shift by 10 turn the number to be negative for star 2. In this way we'd avoid to limit us to single digits, and would only limit to positive numbers.